XCUITests/FxScreenGraph.swift

```diff
@@ -158,11 +158,13 @@ class Action {
     static let ToggleHistoryInNewTab = "ToggleHistoryInNewTab"
 
     static let SelectNewTabAsBlankPage = "SelectNewTabAsBlankPage"
+    static let SelectNewTabAsFirefoxHomePage = "SelectNewTabAsFirefoxHomePage"
     static let SelectNewTabAsBookmarksPage = "SelectNewTabAsBookmarksPage"
     static let SelectNewTabAsHistoryPage = "SelectNewTabAsHistoryPage"
     static let SelectNewTabAsCustomURL = "SelectNewTabAsCustomURL"
 
     static let SelectHomeAsBlankPage = "SelectHomeAsBlankPage"
```

Comment:
This could be removed then, right?

Reply:
Yes I will remove references to both SelectHomeAsBlankPage :) I will leave for NewTab since it applies to new tab.